std/float.glu

```diff
@@ -1,3 +1,5 @@
+//@NO-IMPLICIT-PRELUDE
+
```

Comment:
lololololol

Reply:
Just making it ugly to make it clearer that it is not a permanent solution :P .

Long term I think we want a way to write attributes which can attach to `let`, `type` as well as top level (like #![no_std]) etc.

Reply:
Yeah, I assumed that was the case.